### DISCUSSION TITLE
Slide header with section and subsection

### DISCUSSION THREAD

**Discussion by fbob** [hakimel/reveal.js#3361] [Q&A] [answered]
Hello,
I would like to reproduce a layout from latex beamer using  revealjs which puts the current section and subsection of a presentation a the top of the slide, the slide title being just underneath. Before each new subsection a new slide is displayed showing the section and subsections titles.

See screenshots below as an example.

Thanks !
Have a nice day.

![image](https://user-images.githubusercontent.com/8211124/218037616-04b75855-4934-4d23-9fae-c2e24a1f58c8.png)

![image](https://user-images.githubusercontent.com/8211124/218037690-9c7bc076-a908-487b-a554-9d2b36d483b6.png)

**Comment by t-fritsch** [comment, marked as answer]
The plugin Simplemenu (https://github.com/Martinomagnifico/reveal.js-simplemenu) made by @Martinomagnifico allows to have the sections title in the Header.

It has the option to customize styles and to add any html you want to it : https://martinomagnifico.github.io/reveal.js-simplemenu/demo-custom.html#/firstsubject

my custom config looks like this (pardon the french 😅 ) : 
<img width="860" alt="image" src="https://github.com/hakimel/reveal.js/assets/10920280/2bb51ab2-bd15-48f4-b0c3-a07fcd2a0e66">
